docs/openapi/components/schemas/common/Workflow.yml

```diff
@@ -0,0 +1,24 @@
+title: Workflow
```

Comment:
This uses the word workflow, in our definition we have a path defined not a workflow, really need to settle on one.

also the instance field, isn't this supposed to be a correlationId?

is the intention of the definition field to link to a specific type of workflow? something like "import avocadoes to US"? or is it supposed to be something more akin to "company A imports avocadoes via port X to Country C"? 
in the example above you have a uuid what does that reference/represent?